src/utils/commits-utils.ts

```diff
@@ -131,3 +133,29 @@ export const showPLRMessage = (plr: PipelineRunKind): string => {
     return 'Releasing';
   }
 };
+
+export const createRepoUrl = (commit: Commit): string | null => {
+  if (commit.repoURL) {
+    return commit.repoURL;
+  }
+  if (commit.repoName && commit.repoOrg) {
+    return `https://github.com/${commit.repoOrg}/${commit.repoName}`;
```

Comment:
Valid only if `commit.gitProvider === 'github'`

Reply:
yes, we have checked for git provider wherever this util is used. I will move that check to the methods itself.